site/docs/downloads/index.md

```diff
@@ -73,20 +73,23 @@ The main Nessie server serves the Nessie repository using the Iceberg REST API a
 === "Docker Image"
 
     Docker images are multiplatform images for amd64, arm64, ppc64le, s390x.
+
     They are available from the following repositories:
-     
+    
     * [GitHub Container Registry](https://github.com/projectnessie/nessie/pkgs/container/nessie-cli):
 
     ```bash
     docker pull ghcr.io/projectnessie/nessie-cli:{{ versions.nessie }}
-    docker run -it ghcr.io/projectnessie/nessie-cli:{{ versions.nessie }}
+    docker create -it --name nessie-cli --network=host -v $HOME/.nessie:/home/default/.nessie ghcr.io/projectnessie/nessie-cli:{{ versions.nessie }}
```

Comment:
Why not just `run` in one command?

Reply:
To reuse the same container. Since it's potentially a container that we want to reuse many times, there is no point in creating one container each time.